data/api.go

```diff
@@ -291,6 +301,14 @@ func (a *API) GetData(res http.ResponseWriter, req *http.Request, vars map[strin
 				payload["history"] = parametersHistory["history"]
 				results["payload"] = payload
 			}
+
+			// Add the basal security profile to the pump settings
+			if results["type"].(string) == "pumpSettings" && basalSecurityProfile != nil {
+				payload := results["payload"].(map[string]interface{})
+				payload["basalsecurityprofile"] = basalSecurityProfile
+				results["payload"] = payload
+			}
```

Comment:
Maybe this could be grouped in the previous if (in order to avoid to cast results["payload"] as map[string]interface{} twice)